frontend/src/translations.json

```diff
@@ -140,6 +142,8 @@
       "no_driver_routes":
         "You have no routes stored in database. You can add a new route by choosing option from the menu.",
       "from_to": "From {from} to {destination}",
+      "drive-added-online-notification": "Drive was successfully added to database",
+      "drive-added-offline-notification": "Drive will be added to database after restoring connection to the internet",
```

Comment:
can we please use snake_case? I can tell that translations are getting a bit messy, but dashes in names are least common

Reply:
Done.

Reply:
Awesome!